src/resources/raids/policestation.ts

```diff
@@ -120,8 +120,8 @@ export const station: Location = {
 		},
 		{
 			type: 'LootChannel',
-			name: 'plaza',
-			display: 'plaza',
+			name: 'District Office',
```

Comment:
names must follow discord channel naming rules (lowercase and use - instead of spaces)
```suggestion
			name: 'district-office',
```